coalib/bearlib/aspects/Smell.py

```diff
@@ -75,21 +81,21 @@ class docs:
 @Smell.subaspect
 class MethodSmell:
     """
-    This aspect detects `code smells` related to methods' and functions
-    definitions in your codebase.
+    Code smells related to a method or function definition.
 
     Method-level code smells indicate poorly defined method and or
     functions (too long method or functions, or functions with too many
     parameters) in your source code.
     """
     class docs:
         example = """
-        # This function has way too many parameters
-
-        def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
-            pass
+        class Dummy
+            def yields_a_lot(foo,bar,baz,fling,flung)
```

Comment:
space missing after comma

also this example is showing methods that do not use any instance variables, which I believe is a different problem to what this aspect is about.